AntiCheat/Lang/en_US.resx

```diff
@@ -121,100 +121,100 @@
     <value>Boss attack spoofing detection</value>
   </data>
   <data name="config_ChatReal" xml:space="preserve">
-    <value>Fake chat message detection</value>
+    <value>Detects when a player attempts to spoof their name as another player's.</value>
   </data>
   <data name="config_DespawnItem" xml:space="preserve">
-    <value>Abnormal item despawn detection</value>
+    <value>Detects when items are unusually destroyed or deleted.</value>
   </data>
   <data name="config_Enemy" xml:space="preserve">
-    <value>Abnormal monster behavior detection</value>
+    <value>Detects when a monster behaves abnormally.</value>
   </data>
   <data name="config_FreeBuy" xml:space="preserve">
-    <value>Free purchases detection</value>
+    <value>Detects players ordering items without spending credits.</value>
   </data>
   <data name="config_Gift" xml:space="preserve">
-    <value>Abnormal gift box interaction detection</value>
+    <value>Detects players having abnormal gift box interactions.</value>
   </data>
   <data name="config_GrabObject" xml:space="preserve">
-    <value>Item pickup distance detection</value>
+    <value>Detects picking up items outside of default grab range.</value>
   </data>
   <data name="config_InfiniteAmmo" xml:space="preserve">
-    <value>Infinite ammo detection</value>
+    <value>Detects when a player has infinite ammo.</value>

```

Comment:
u need to know, if the shotgun already loaded infinite ammo, normal player can also trigger that detection

Reply:
I will address this and the rest of the problems you listed momentarily. Thank you.